drastic-price-change-template/bot-config.json.example

```diff
@@ -0,0 +1,8 @@
+{
+  "priceDiscrepancyThreshold": 10,
+  "asset": {
+    "contract": "0x1f9840a85d5af5bf1d1762f925bdaddc4201f984",
```

Comment:
what is this?

Reply:
This is the token contract. We use it to get all transfers from/to it. In this case this is the UNI token